source/includes/document-filtering-limitations.rst

```diff
@@ -0,0 +1,27 @@
+- ``field`` names can't contain dots (``.``) or dollar signs (``$``).
+
+- ``matchValues`` array items can't be any of the following BSON types: 
+
+  - Arrays 
+  - Embedded documents 
+  - Regular expressions
```

Comment:
This list is actually longer.

It’d be better to list the types that _can_ be in the filter:
- all number types
- strings & binary
- boolean
- objectID
- datetime